_posts/2016-06-20-alpha-release-notes.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "argparse2cwl alpha release notes"
+date:   2016-06-20 02:27:27 +0300
+comments: true
+---
+
+`argparse2cwl` forms CWL tool descriptions from programs which use `argparse` as their argument parser. 
+
+## Prerequisites ##
+`argparse2cwl` is compatible with both Python 2 & 3.
+Dependencies:
+* Jinja2 >= 2.8
+
+## Installing ##
+
+Installation process:
+
+1. Clone the source repository:
```

Comment:
@brainstorm The only stable installation process I found doesn't require PyPi: https://github.com/anton-khodak/argparse2cwl-blog/pull/1#issuecomment-229892248. Installing via `pip` requires setting `PYTHONPATH` for each session, that's unpractical.